PENDING.md

```diff
@@ -39,7 +39,13 @@ BREAKING CHANGES
 * [types] sdk.NewCoin now takes sdk.Int, sdk.NewInt64Coin takes int64
 * [cli] #1551: Officially removed `--name` from CLI commands
 * [cli] Genesis/key creation (`init`) now supports user-provided key passwords
-* [cli] [#2014](https://github.com/cosmos/cosmos-sdk/pull/2014) `gaiacli advanced` no longer exists - to access `ibc`, `rest-server`, and `validator-set` commands use `gaiacli ibc`, `gaiacli rest-server`, and `gaiacli tendermint`, respectively
```

Comment:
For future reference - there is no need to include the github PR link - the issue number/PR should be enough for anyone to look up the PR/issue on github

Reply:
Sorry @rigelrozanski I've been doing this as I found it handy as other major projects do it as well.